Diagnostics/HealthChecker/Features/Get-LoadBalancingReport.ps1

```diff
@@ -0,0 +1,324 @@
+﻿# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+. $PSScriptRoot\..\Helpers\PerformanceCountersFunctions.ps1
+. $PSScriptRoot\..\..\..\Shared\Invoke-CatchActionErrorLoop.ps1
+function Get-LoadBalancingReport {
+    Write-Verbose "Calling: $($MyInvocation.MyCommand)"
+    $CASServers = @()
+    $MBXServers = @()
+
```

Comment:
Right here let's do `$getExchangeServer = Get-ExchangeServer`

This way we aren't doing multiple `Get-ExchangeServer`.

Or you do a single `Get-ExchangeServer` and do a foreach loop and add the server to a list if everything is true. 

For large environments, we don't want to run `Get-ExchangeServer` twice if we can avoid it.

Reply:
Fixed